### DISCUSSION TITLE
Values do not match in e3nn.nn.Activation

### DISCUSSION THREAD

**Discussion by skiyohara** [e3nn/e3nn#499] [General]
Dear all, 

I am about to use e3nn.nn.Activation, but it behaves a little strangely.

Here is a sample:
```
irreps_in = Irreps("6x0o+2x1e")
a = e3nn.nn.Activation("6x0o+2x1e", [torch.abs, None])

x = irreps_in.randn(1,-1,dtype=torch.float64)
ax = a(x)
```

I expect the absolute function (torch.abs) to act on the scalars corresponding to 6x0o. While negative values are transformed into positive values as expected, the magnitudes are slightly changed (even for values that were originally positive). A direct application of the absolute function preserves the original magnitudes exactly.

Does anyone know how to solve this issue?
Any suggestions would be greatly appreciated.

**Comment by mitkotak** [comment]
Yes this is because we normalize the activation output by a constant [here](https://github.com/e3nn/e3nn/blob/f95297952303347a8a3cfe971efe449c710c43b2/e3nn/math/_normalize_activation.py#L57) before returning it

**Comment by skiyohara** [comment]
Thank you mitkotak.
I read the source code of the Activation function, but I overlooked some details.

I appreciate it again.

**Reply by mitkotak** [reply]
Yeah sorry for not documenting this well. At first even I thought this was some kind of bug :sweat_smile: but then checked that we also do this in [jax](https://github.com/e3nn/e3nn-jax/blob/6f6319d2b72e265d38814ebfae76bfe2c9688b96/e3nn_jax/_src/activation.py#L59-L61) version
